.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
+*   @opensearch-project/opensearch-dashboards-core
```

Comment:
Is this true? Or should we have a separate team?

Reply:
We have an `oui` team; i will switch to that.